android/src/main/kotlin/io/customer/customer_io/messaginginapp/FlutterInAppPlatformDelegate.kt

```diff
@@ -17,15 +17,19 @@ class FlutterInAppPlatformDelegate(
     private val methodChannel: MethodChannel
 ) : WrapperPlatformDelegate(view) {
 
-    companion object {
-        private const val TAG = "FlutterInAppPlatformDelegate"
-    }
-
     /**
      * Flutter-specific event dispatch implementation.
      * This is the ONLY platform-specific code - everything else is now shared!
      */
     override fun dispatchEvent(eventName: String, payload: Map<String, Any?>) {
         methodChannel.invokeMethod(eventName, payload)
     }
+    
+    /**
+     * Public method for external classes to dispatch events.
```

Comment:
It should be internal I believe?

```suggestion
     * Public method for internal classes to dispatch events.
```